src/components/atoms/Date/Date.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import * as dayjs from 'dayjs';
+
+const DEFAULT_FORMAT = 'DD MM YYYY hh:mm';
+
+export const Date = ({ format = DEFAULT_FORMAT, children }) => {
+    const formatedDate = dayjs(children).format(format);
```

Comment:
Such a overkill, we should just create helper function to parse date which will accept the two parameters
date, format, with default format
and return formatted date, we really don't need the separate component for that, overkilllll